src/components/Mission.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { leaveMission, joinMission } from '../redux/missions/missionReducer';
+
+const Mission = (props) => {
+  const {
+    id,
+    name,
+    description,
+    join,
+  } = props;
+  const dispatch = useDispatch();
+  const handleClick = () => {
+    if (join) {
+      dispatch(leaveMission(id));
+    } else {
+      dispatch(joinMission(id));
+    }
+  };
```

Comment:
Actions should only be dispatched once and they are being dispatched twice, please fix this.

<img width="305" alt="Screen Shot 2022-08-26 at 8 17 41" src="https://user-images.githubusercontent.com/82426682/186912390-612790ab-5d8b-44dc-94ba-89dafb2ba69c.png">

<img width="218" alt="Screen Shot 2022-08-26 at 8 18 10" src="https://user-images.githubusercontent.com/82426682/186912478-dcf7a779-912e-463e-80e8-f50f473c9510.png">